spec/lib/terraforming/resource/ebenv_spec.rb

```diff
@@ -0,0 +1,140 @@
+            date_created: Time.parse("2016-06-03 13:18:00 UTC"),
+            date_updated: Time.parse("2016-06-29 08:57:53 UTC"),
+            deployment_status: "deployed", 
+            description: "Description of hoge", 
+            environment_name: "hoge-env", 
+            option_settings: [
+                {
+                  resource_name: "AWSEBAutoScalingGroup", namespace: "aws:autoscaling:asg", option_name: "Availability Zones", value: "Any"
+                },
+                {
+                  resource_name: "AWSEBAutoScalingGroup", namespace: "aws:autoscaling:asg", option_name: "MinSize", value: "1"
+                },
+                {
+                  resource_name: "AWSEBAutoScalingLaunchConfiguration", namespace: "aws:autoscaling:launchconfiguration", option_name: "RootVolumeIOPS", value: ""
+                }
+             ], 
+            solution_stack_name: "64bit Amazon Linux 2016.03 v2.1.1 running Tomcat 8 Java 8", 
+          }, 
+        ]
+      end
+
+      before do
+        client.stub_responses(:describe_environments, environments: environments)
+        client.stub_responses(:describe_configuration_settings, configuration_settings: hoge_settings )
+      end
+
+      describe ".tf" do
+        it "should generate tf" do
+          expect(described_class.tf(client: client)).to eq <<-EOS
+resource "aws_elastic_beanstalk_environment" "hoge-env" {
+  name = "hoge-env"
+  application = "hoge"
+  solution_stack_name = "64bit Amazon Linux 2016.03 v2.1.1 running Tomcat 8 Java 8"
+  description = "Description of hoge"
+  cname_prefix = "hoge-env"
+  tier = "WebServer"
+
+  setting {
+    namespace = "aws:autoscaling:asg"
+    name      = "Availability Zones"
+    value     = "Any"
+    resource  = "AWSEBAutoScalingGroup"
+  }
+
+  setting {
+    namespace = "aws:autoscaling:asg"
+    name      = "MinSize"
+    value     = "1"
+    resource  = "AWSEBAutoScalingGroup"
+  }
+
+  setting {
+    namespace = "aws:autoscaling:launchconfiguration"
+    name      = "RootVolumeIOPS"
+    value     = ""
+    resource  = "AWSEBAutoScalingLaunchConfiguration"
+  }
+}
+
+        EOS
+        end
+      end
+
+      describe ".tfstate" do
+        it "should generate tfstate" do
+          expect(described_class.tfstate(client: client)).to eq({
+            "aws_elastic_beanstalk_environment.hoge-env" => {
+              "type" => "aws_elastic_beanstalk_environment",
+              "primary" => {
+                "id" => "hoge-env",
+                "attributes" => {
+                  "name" => "hoge-env",
+                  "description" => "Description of hoge",
+                  "application" => "hoge",
+                  "cname_prefix" => "hoge-env",
+                  "tier" => "WebServer",
+                  "solution_stack_name" => "64bit Amazon Linux 2016.03 v2.1.1 running Tomcat 8 Java 8",
+                  "settings.#" => "3",
+                  "setting.3844072155.namespace"=>"aws:autoscaling:asg",
```

Comment:
__[rubocop]__
* Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->